h5py/tests/test_dataset.py

```diff
@@ -713,6 +796,14 @@ def test_track_times(self):
         similar = self.f.create_dataset_like('lenovo', orig)
         self.assertEqual(0, h5py.h5g.get_objinfo(similar._id).mtime)
 
+    def test_special_case(self):
```

Comment:
```suggestion
    def test_maxshape(self):
```

'Special case' is (ironically) a very generic term.